src/main/java/org/broadinstitute/hellbender/tools/variantdb/arrays/ArrayExtractCohort.java

```diff
@@ -98,6 +98,12 @@
     )
     private String cohortTable = null;
 
+    @Argument(
+            fullName = "gt-only",
+            doc = "If true, only get the genotype info",
```

Comment:
Can you say in the doc here what is excluded since it's not that many fields (NORMX, NORMY, LLR, and BAF)